### DISCUSSION TITLE
multi vector models?

### DISCUSSION THREAD

**Discussion by raghavlite** [embeddings-benchmark/mteb#1481] [Q&A]
Is there a example to use multi vector models like Colbert?

**Comment by KennethEnevoldsen** [comment]
(Moving this to discussion as it is a usage question.)

I believe this might be what you are looking for:
https://github.com/embeddings-benchmark/mteb?tab=readme-ov-file#using-a-cross-encoder-for-reranking

**Comment by sam-hey** [comment]
I've raised this issue: https://github.com/embeddings-benchmark/mteb/issues/1560. At the moment, MTEB only supports cosine similarity and dot product, which means it doesn't accommodate Multi-Vector Models like ColBERT. I hope this limitation can be addressed soon.

Best regards
Sam

**Reply by CZH-THU** [reply]
Hi,I saw the commits of #1563,but the main branch still can't support ColBert?

**Reply by Samoed** [reply]
Yes, main doesn't support

**Reply by CZH-THU** [reply]
Will it support in the future?

**Reply by sam-hey** [reply]
The PR is approved and is expected to be merged soon. Please note that the current implementation does not utilize any indexing, making it resource-intensive for large datasets. This will be updated once Pylate introduces the PLAID Index.

**Reply by CZH-THU** [reply]
Nice! Thanks.

**Reply by Samoed** [reply]
I've merged this PR to main
